app/main.py

```diff
@@ -1,8 +1,42 @@
 class Car:
-    # write your code here
-    pass
-
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int, average_rating: float, count_of_ratings: int):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car):
+        washing_price = round((car.comfort_class * (self.clean_power - car.clean_mark) * self.average_rating / self.distance_from_city_center), 1)
+        return washing_price
+
+    def serve_cars(self, cars: list[Car]):
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+                print(self.calculate_washing_price(car))
```

Comment:
The method `calculate_washing_price` is called twice for each car: once to add to `income` and once to print the price. Consider storing the result in a variable to avoid redundant calculations.